src/pages/resultsView/mutualExclusivity/MutualExclusivityUtil.tsx

```diff
@@ -10,15 +10,18 @@ export function calculateAssociation(logOddsRatio: number): string {
     return logOddsRatio > 0 ? "Co-occurrence" : "Mutual exclusivity";
 }
 
-export function countOccurences(valuesA: boolean[], valuesB: boolean[]): [number, number, number, number] {
+export function countOccurences(valuesA: (boolean | undefined)[], valuesB: (boolean | undefined)[]): [number, number, number, number] {
 
     let neither = 0;
     let bNotA = 0;
     let aNotB = 0;
     let both = 0;
 
     valuesA.forEach((valueA, index) => {
-
+        // jump over this comparison if there is undefined value(not profiled value)
```

Comment:
@dippindots i don't see a test that exercises this new logic.  would seem like it would have to assert against the counts returned by this function.

Reply:
@alisman Yes, I should add the tests for this function, will update soon.